src/main/java/com/axway/apim/swagger/APIImportConfigAdapter.java

```diff
@@ -109,6 +113,18 @@ public IAPI getDesiredAPI() throws AppException {
 			} else {
 				stagedConfig = baseConfig;
 			}
+			Path currentRelativePath = Paths.get("");
+			String s = currentRelativePath.toAbsolutePath().toString();
+			LOG.info("path={}",s);
+			if (StringUtils.isEmpty(this.pathToAPIDefinition)) {
+				if (StringUtils.isNotEmpty(stagedConfig.getApiDefinitionImport())) {
+					this.pathToAPIDefinition=baseConfig.getApiDefinitionImport();
+					LOG.info("Reading API Definition from contract file");
+				} else {
+					throw new AppException("No API Definition configured", ErrorCode.NO_API_DEFINITION_CONFIGURED);

```

Comment:
Please provide the logStackTrace parameter with false, to avoid having the StackTrace logged on this error.

Reply:
ok